ci/tests/small_fixed_alloc.c

```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
```

Comment:
Is this `clang-format`ted?

Reply:
Have now clang-formatted this test code in [2cc9b31d](https://github.com/capablevms/bdwgc/pull/11/commits/2cc9b31d999e78ceced82eadae95bcd57306d72a). However, boehmGC is not generally formatted using clang-formatted.